src/OSPSuite.SimModelSolver_CVODES/OSPSuite.SimModelSolver_CVODES_Ubuntu18.nuspec

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <description>SimModel Solver</description>
+    <id>OSPSuite.SimModelSolver_CVODES.Ubuntu18</id>
```

Comment:
Linux nuget package has `.Ubuntu18` suffix. Going to rename FuncParser Linux nuget package as well.